catalog/models.py

```diff
@@ -21,6 +21,10 @@ class Language(models.Model):
     """Model representing a Language (e.g. English, French, Japanese, etc.)"""
     name = models.CharField(max_length=200,
                             help_text="Enter the book's natural language (e.g. English, French, Japanese etc.)")
+    
+    def get_absolute_url(self):
+        """Returns the url to access a particular language books."""
+        return reverse('laguage-books', args=[str(self.id)])
```

Comment:
Thanks!
1. typographic error in laguage-books - "language". 

This isn't enough "in itself" to list all books in a language, and is not actually needed to do so. How would this actually work?

My concern here is that this example is associated with the MDN documentation, which is carefully designed as an example, showing just enough. So to accept this we have to have corresponding documentation updates on MDN and also be sure that it makes the example better - ie shows new things. It is not enough that it makes the library do new things.

Reply:
Thank to you for your time.
I just starting in this and only want to participate.

Reply:
Your participation is very welcome. 

I hope my comment is not putting you off. It is just important to understand that we could have created a beautiful and fully functional library, but then it would be hard to maintain, hard for readers to see "what is important" etc. 

So feel free to make these changes in your own code, and if you think they show something different that is not shown by the other code very happy to take them.

For now though I will close this proposal.